File: Go/alg10.go
// ALG10
package main
import ("fmt"; "bufio"; "os")
func main() {
  reader := bufio.NewReader(os.Stdin)
  text, _ := reader.ReadString('\n')
  var X int
  fmt.Sscanf(text, "%d", &X)
  text, _ = reader.ReadString('\n')
  var Y int
  fmt.Sscanf(text, "%d", &Y)
  var N1 int
  var N2 int
  if X > Y {
    N1 = Y
    N2 = X
  } else {
    N1 = X
    N2 = Y
  }
  fmt.Printf("%d\n", N1)
  fmt.Printf("%d\n", N2)
}

File: Dart/alg10.dart
/* ALG10 */
import 'dart:io';
void main() {
  int X;
  int Y;
  int N1;
  int N2;
  X = int.parse(stdin.readLineSync()!);
  Y = int.parse(stdin.readLineSync()!);
  if (X > Y) {
    N1 = Y;
    N2 = X;
  } else {
    N1 = X;
    N2 = Y;
  }
  print('$N1');
  print('$N2');
}
